### PR TITLE
add py39 to tox tests

### PR DESCRIPTION
**Pull request by wpoely86**
Let's be future proof and start testing python 3.9 also?

### CODE REVIEW

File: lib/vsc/install/shared_setup.py [hunk 1]
```diff
@@ -291,7 +291,7 @@ def get_egg_cache_dir_pyver(self):
 
         # the original get_egg_cache_dir creates the .eggs directory if it doesn't exist yet,
         # but we want to have it versioned, so we rename it
-        egg_cache_dir_pyver = egg_cache_dir + '.py%s' % sys.version_info[0]
+        egg_cache_dir_pyver = egg_cache_dir + '.py%s' % '.'.join([str(x) for x in sys.version_info[0:2]])
```
Comment (wdpypere) [inline lib/vsc/install/shared_setup.py:294]: is this strictly needed?

**Comment by wdpypere** [comment]
@wpoely86 you will need https://github.com/hpcugent/vsc-install/pull/229 to make this build.

File: lib/vsc/install/shared_setup.py (hunk 1 above)
Comment (wpoely86) [reply lib/vsc/install/shared_setup.py:294]: Maybe not but it's a lot cleaner not to mix all python3 versions.

I guess it's not needed if you have a clean directory every time.

File: lib/vsc/install/ci.py [hunk 2]
```diff
@@ -146,8 +146,8 @@ def gen_tox_ini():
     envs = []
 
     # always run tests with Python 3
-    py3_env = 'py36'
-    envs.append(py3_env)
+    py3_envs = ['py36', 'py39']
```
Comment (stdweird) [inline lib/vsc/install/ci.py:149]: how do we run tests without 39 in certain repos?

Comment (wpoely86) [reply lib/vsc/install/ci.py:149]: We can add it to `vsc-cfg.ini` to optionally disable it?

**Comment by wpoely86** [comment]
I will revamp this one when #229 is merged

File: lib/vsc/install/ci.py (hunk 2 above)
Comment (wdpypere) [reply lib/vsc/install/ci.py:149]: might be a good idea, same way we did it for the py3 move.

Comment (wdpypere) [reply lib/vsc/install/ci.py:149]: I'll look into it

Comment (stdweird) [reply lib/vsc/install/ci.py:149]: yes, should be dealt with same as in py3 move: first it's opt in, then mandatory with optout

File: test/ci.py [hunk 3]
```diff
@@ -107,6 +107,9 @@
 EXPECTED_TOX_INI_PY36_IGNORE = """
 [testenv:py36]
 ignore_outcome = true
+
+[testenv:py39]
+ignore_outcome = true
```
Comment (wdpypere) [inline test/ci.py:112]: this does not work right? the outcomes are not ignored

File: lib/vsc/install/ci.py (hunk 2 above)
Comment (wdpypere) [reply lib/vsc/install/ci.py:149]: OK, I'll get something working

**Comment by wdpypere** [comment]
superseded by https://github.com/hpcugent/vsc-install/pull/230

**Pull request closed without merging**